examples/extensions/add_prices.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example adds a price extension and associates it with an account.
+
+Campaign targeting is also set using the specified campaign ID. To get
+campaigns, run basic_operations/get_campaigns.py
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.ads.google_ads.v2.services.campaign_service_client import (
+    CampaignServiceClient
+)
+
+
+def main(client, customer_id, campaign_id):
+    """The main method that creates all necessary entities for the example.
```

Comment:
I hate to be nitpicky, but we have historically excluded pydocs from simple methods in examples where their purpose is intuitively apparent. Most existing examples don't do this, but is there a push to change this going forward?

Reply:
was trying to make pylint happy. removed.